requirements/test.txt

```diff
@@ -113,3 +112,6 @@ virtualenv==16.6.0        # via tox
 wcwidth==0.1.7            # via prompt-toolkit
 whitenoise==3.3.0
 wrapt==1.11.1
+
+# The following packages are considered to be unsafe in a requirements file:
```

Comment:
I think this has been discussed already here https://github.com/artefactual/archivematica-storage-service/pull/473#discussion_r297287410